src/vs/vscode.proposed.d.ts

```diff
@@ -945,16 +945,10 @@ declare module 'vscode' {
 		Code = 2
 	}
 
-	export enum NotebookCellRunState {
-		Running = 1,
-		Idle = 2,
-		Success = 3,
-		Error = 4
-	}
-
-	export enum NotebookRunState {
-		Running = 1,
-		Idle = 2
+	export interface NotebookCellPreviousExecutionResult {
```

Comment:
Maybe a better name is `NotebookCellExecutionSummary`

Reply:
Do you like `previousResult` as the property name on the cell or something else?